Spigot-Server-Patches/0236-Implemented-the-cancellable-ban-events.patch

```diff
@@ -0,0 +1,131 @@
+From 996f744b89e05a99e74d5768d13363cfb8975a41 Mon Sep 17 00:00:00 2001
+From: Sam Ezeh <spammy23@users.noreply.github.com>
+Date: Wed, 30 Aug 2017 20:34:26 +0100
+Subject: [PATCH] Implemented the cancellable ban events
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftIpBanList.java b/src/main/java/org/bukkit/craftbukkit/CraftIpBanList.java
+index 80832f78..05791767 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftIpBanList.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftIpBanList.java
+@@ -1,19 +1,19 @@
+ package org.bukkit.craftbukkit;
+ 
+-import java.io.IOException;
```

Comment:
Don't reorganise imports, you might wanna disable this feature in your IDE for dealing with paper stuff as it causes issues like this that make handling diffs a headache for everyone.